libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/eh/EHorizonMapper.kt

```diff
@@ -137,6 +139,8 @@ internal fun RoadObjectDistanceInfo.mapToRoadObjectDistanceInfo(
             )
         }
         isSubGraphDistanceInfo -> with(subGraphDistanceInfo) {
+            val distanceToNearestEntry = if (entrances.isNotEmpty()) entrances[0].distance else 0.0
+            val distanceToNearestExit = if (exits.isNotEmpty()) exits[0].distance else 0.0
```

Comment:
Same as above: @korshaknn @mskurydin @etl @SiarheiFedartsou Do we want to expose `entrances` and `exits`? 🤔

Reply:
Do we want then to expose publicly `SubGraphDistanceInfo` entirely?

Reply:
I think yes, all the fields from nav-native api should be exposed